validate.js

```diff
@@ -0,0 +1,11 @@
+const Form = document.querySelector('.form');
+const emailInput = document.querySelector('.email');
+const validation = document.querySelector('.message-validation');
+
+Form.addEventListener('submit', (event) => {
+  const stringMessage = emailInput.value;
+  if (/[A-Z]/.test(stringMessage)) {
+    validation.innerHTML = 'Error! Your submission was unsuccessful, please input email in lowercase.';
+    event.preventDefault();
```

Comment:
- [ ] Great job so far, one of the project requirements is that when a user submits an email containing an uppercase letter it should show an error message above the button, I can submit with an uppercase email address, please fix that. The position of the preventDefault plays an important role.

Reply:
hi @Wes-Isaac , the review changes have been made and ready for your review,...thank you 😄